src/arm/BONE-UART3-00A0.dts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2020 Deepak Khatri <deepaklorkhatri7@gmail.com>
+ *
+ * Virtual cape for /bone/uart/3 (only on BBB)
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+ 
+/dts-v1/;
+/plugin/;
+
+&{/chosen} {
+    overlays {
+        BONE-UART3-00A0 = __TIMESTAMP__;
+    };
+};
+
+&ocp {
+    P9_42_pinmux { status = "disabled"; };
+};
+
+&bone_uart_3 {
+    status = "okay";
```

Comment:
Seems to me like this would be a good place for the symlink.

Reply:
I have added the symlinks in this file -> https://github.com/beagleboard/BeagleBoard-DeviceTrees/pull/17/commits/cafd17132522e4c73a6d923914b20c8434f85aa2

so that people don't have to remember to add symlinks every time they want to use any bone bus, should I remove move them here?

Reply:
Makes sense to have the symlinks and pinmuxes in the main file based on this. Please make sure to reference what is needed in the base tree.